### PR TITLE
Dev/mar bi/57 card container

### PR DESCRIPTION
**Pull request by mar-bi**
Happy to introduce `CardContainer`.

A lot of files changed because I need it to work properly with `StoryCard` and `ProjectCard`.
Hope you will like it.  
**Bonus:** building pages is more easy now. Some amount of work is done. :upside_down_face:

### CODE REVIEW

File: src/layouts/components/ProjectCard.js [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 import styled from 'styled-components'
 
 import { StyledH3, StyledP, StyledUl, StyledLi } from '../../theme/globalStyle'
 import { ButtonSmall } from '../components/Button'
 
+import faker from 'faker'
+
 const Wrapper = styled.div`
-  width: 400px;
+  width: ${props => props.width || 'auto'};
```
Comment (spences10) [inline src/layouts/components/ProjectCard.js]: 👌

File: src/layouts/components/ProjectCard.js [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
   display: grid;
-  grid-template-columns: repeat(1fr);
-  grid-template-rows: auto;
+  grid-template-columns: 1fr;
+  grid-template-rows: minmax(100px, auto);
```
Comment (spences10) [inline src/layouts/components/ProjectCard.js]: What does this do?

File: src/pages/index.js [hunk 3]
```diff
@@ -83,9 +83,9 @@ const IndexPage = ({ data }) => (
       <h3>Hello world!</h3>
     </Divider>
     <Divider justify={'space-around'}>
-      <Icon icon={ICONS.RULER} size={50} color={'white'} />
-      <Icon icon={ICONS.BLOG} size={50} color={'white'} />
-      <Icon icon={ICONS.RULER} size={50} color={'white'} />
+      <Icon icon={ICONS.RULER} size={50} color={props.theme.white} />
```
Comment (spences10) [inline src/pages/index.js]: 👍

**Comment by spences10** [comment]
Closes #57

File: src/layouts/components/ProjectCard.js (hunk 2 above)
Comment (mar-bi) [reply src/layouts/components/ProjectCard.js]: it's a css function for the grid. 
here it defines a size range for the row. The row must be greater than or equal to min (first value) and less than or equal to max(last value).

Comment (mar-bi) [reply src/layouts/components/ProjectCard.js]: Actually, i find it in the video you suggested for **css grid** :smile:

File: src/pages/index.js (hunk 3 above)
Comment (mar-bi) [reply src/pages/index.js]: :smile: 
I caught it via the browser console.

**Pull request merged**